modules/drivers/ili9341/modIli9341.c

```diff
@@ -534,6 +545,22 @@ void ili9341Begin(void *refcon, CommodettoCoordinate x, CommodettoCoordinate y,
 
 	xMin = x + MODDEF_ILI9341_COLUMN_OFFSET;
 	yMin = y + MODDEF_ILI9341_ROW_OFFSET;
+	
+	switch(sd->rotation) {
+		case 1:
+			// 90 degrees
+			yMin += ILI9341_GRAM_X_OFFSET;
+			break;
+		case 2:
+			// 180 degrees
+			yMin += ILI9341_GRAM_Y_OFFSET;
+			xMin += ILI9341_GRAM_X_OFFSET;
+			break;
+		case 3:
+			// 270 degrees
+			xMin += ILI9341_GRAM_Y_OFFSET;
```

Comment:
The last `case` in the `switch` should have a `break` for consistency and safety.